web/src/main/java/edu/softserve/zoo/controller/rest/impl/EmployeeRestController.java

```diff
@@ -31,4 +34,10 @@ public EmployeeRestController() {
     protected Service<Employee> getService() {
         return employeeService;
     }
+
+    @Override
+    @RequestMapping(method = RequestMethod.GET)
+    public List<EmployeeDto> getAll() {
+        return super.getAll();
```

Comment:
I think i haven't got your point. It's just controller method for GET on /employees endpoint. There will be a requirement to list all existing employees.